pkgs/development/python-modules/cachy/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, buildPythonPackage, fetchPypi
+, redis
+, memcached
+, msgpack-python
+}:
+buildPythonPackage rec {
+  pname = "cachy";
+  version = "0.2.0";
+
+  src = fetchPypi {
```

Comment:
:rofl: No tests in here either, don't get why people aren't distributing these things.

I'm flipping the switch and saying that we should probably use `fetchFromGitHub` to get a source archive for all packages that don't have tests on pypi. Only exception would be when the tests look problematic.

Reply:
Also want's `poetry`.